spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -35,35 +35,32 @@ The specifications for each release are listed in reverse chronological order.
-* https://download.eclipse.org/microprofile/microprofile-open-api-3.1/microprofile-openapi-spec-3.1.html#_incompatible_changes[OpenAPI]
-* https://download.eclipse.org/microprofile/microprofile-rest-client-3.0.1/microprofile-rest-client-spec-3.0.1.html#_incompatible_changes[Rest Client]
-* https://download.eclipse.org/microprofile/microprofile-jwt-auth-2.0/microprofile-jwt-auth-spec-2.0.html#_incompatible_changes[JWT] (TODO - Update to JWT 2.1?)
-* https://download.eclipse.org/microprofile/microprofile-telemetry-1.0/microprofile-telemetry-spec-1.0.html#_incompatible_changes[Telemetry]
-
+If you are still dependent on Jakarta EE 9.1, please consider using the https://github.com/eclipse/microprofile/releases/tag/5.0[5.0 release of MicroProfile].
 If you are still dependent on Jakarta EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/4.1[4.1 release of MicroProfile].
 If you are still dependent on Java EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/3.3[3.3 release of MicroProfile].
 If you are still dependent on Java EE 7, please consider using the https://github.com/eclipse/microprofile/releases/tag/1.4[1.4 release of MicroProfile].
 
 The complete list of MicroProfile 6.0 specifications includes:
 
+ - https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE 10 Core Profile]
  - https://github.com/eclipse/microprofile-config/releases/tag/3.0.2[MicroProfile Config 3.0]
- - https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/4.0.1[MicroProfile Fault Tolerance 4.0]
+ - https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/4.0.2[MicroProfile Fault Tolerance 4.0]
  - https://github.com/eclipse/microprofile-metrics/releases/tag/5.0[MicroProfile Metrics 5.0]
  - https://github.com/eclipse/microprofile-health/releases/tag/4.0.1[MicroProfile Health 4.0]
  - https://github.com/eclipse/microprofile-telemetry/releases/tag/1.0[MicroProfile Telemetry 1.0]
  - https://github.com/eclipse/microprofile-open-api/releases/tag/3.1[MicroProfile OpenAPI 3.1]
- - https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.0[MicroProfile JWT Authentication 2.0] (TODO: Update to 2.1?)
+ - https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.1[MicroProfile JWT Authentication 2.1]
  - https://github.com/eclipse/microprofile-rest-client/releases/tag/3.0.1[MicroProfile Rest Client 3.0]
- - https://jakarta.ee/specifications/coreprofile/10/[Jakarta JSON Processing 2.0]
+
+MicroProfile 6.0 has the following backward incompatible changes compared to MicroProfile 5.0.
+
+* https://download.eclipse.org/microprofile/microprofile-metrics-5.0.0/microprofile-metrics-spec-5.0.0.html#_incompatible_changes[MicroProfile Metrics Incompatible changes]
+* MicroProfile OpenTracing replaced by MicroProfile Telemetry
+* Based on Jakarta EE 10 Core Profile instead of the five specifications in Jakarta EE 9.1
```

Comment:
Seen the discussion we're having on the mailing list, is "based" the correct wording here? It if clear to me, but it must also be to others. And let's use that terminology in future releases then

Reply:
I think this is accurate word for me as well. We can update this if someone comes up a better one.

Reply:
The term "Based on" is vague in its meaning.  More precise wording would be "Includes the ..." to be consistent with the section, "The complete list of MicroProfile 6.0 specifications includes..."
But even we leave it as the more vague "based on", the presence of that of later statement should clarify the intent.